jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.jar;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.jar.JarFile;
+
+public class JarProcessor {
+
+  /**
+   * Jar Type.
+   *
+   * <ul>
+   *   <li>{@code REGULAR} a regular jar.
+   *   <li>{@code SPRING_BOOT} a springboot fat jar.
+   * </ul>
+   */
+  public enum JarType {
+    REGULAR,
```

Comment:
I think `STANDARD` is probably the more standard term?

Reply:
Done.